pc_min.m

```diff
@@ -0,0 +1,388 @@
+%
+          function [PMIN,VMAX,TO,LNB,IFL]= pc_min(SST,PSL,P,T,R)
+%
+%   Revised on 9/24/2005 to fix convergence problems at high pressure
+%     Converted to MATLAB  5/17/2008
+%
+%   Revised 7/20/15 by D. Gilford to output the LNB
```

Comment:
If i'm not mistaken there was another version of pc_min that Kerry uploaded that fixed recently: 
%   Revised again 1/7/2017 by Luke Davis to fix and improper definition of
%   JMIN, which should be the lowest profile level at or above the parcel
%   level in the calculation of CAPE. I am grateful to Luke and to Tim
%   Merlis for pointing out this error. In practice, the error was
%   usually less than 1 hPa in central pressure.

Reply:
Yes! As you noted below, I added these to pi.py.